### ISSUE TITLE
Local moran with NA

### ISSUE THREAD

**Issue by sayala-spepi** [GeoDaCenter/rgeoda#45]
How Geoda calculates the local moran with NA values, because Rgeoda does not allow it because it cannot calculate with NA. What extra process does Geoda have than Rgeoda?

**Comment by lixun910** [comment]
I think it will take care of the NA values under the hood: see https://github.com/GeoDaCenter/rgeoda/blob/94c17daf55f7da1ebb32f78516683bb9dfd7b444/src/rcpp_lisa.cpp#L129 so there is no extra process needed.

**Comment by sayala-spepi** [comment]
yes, but in line 141 undefs[i] = data.is_na(i); treats them to leave them out. In Rgeoda there is no line to treat the NA. Maybe that can make a difference
